api/qanpb/object_details.proto

```diff
@@ -50,6 +50,7 @@ message MetricsRequest {
 // MetricsReply defines metrics for specific value of dimension (ex.: host=hostname1 or queryid=1D410B4BE5060972.
 message MetricsReply {
   map<string, MetricValues> metrics = 3;
+  map<string, string> text_metrics = 7;
```

Comment:
text_metrics? what does it include?

Reply:
@BupycHuk  top_query and query_plan for now.